pkg/cmd/pr/shared/commentable.go

```diff
@@ -260,6 +311,12 @@ func CommentableEditSurvey(cf func() (gh.Config, error), io *iostreams.IOStreams
 	}
 }
 
+func CommentableConfirmDeleteComment(p Prompt) func(string) (bool, error) {
+	return func(body string) (bool, error) {
+		return p.Confirm(fmt.Sprintf("Deleting the comment with a body: '%s'.", body), true)
```

Comment:
I think the prompt format reads better as `"Delete the comment: %s?"`

However, multiline/long comments might create a unpleasant experience here. I'll put a comment down here to discuss it separately.